src/main/java/net/mcreator/ui/dialogs/file/FileDialogs.java

```diff
@@ -140,7 +140,7 @@ private static FileChooser.ExtensionFilter[] getFileFiltersForStringArray(String
 	}
 
 	private static boolean useNativeFileChooser() {
-		return PreferencesManager.PREFERENCES.ui.useNativeFileChooser.get() && OS.getOS() == OS.WINDOWS;
+		return PreferencesManager.PREFERENCES.ui.useNativeFileChooser.get() && OS.getOS() != OS.LINUX;

```

Comment:
PreferencesManager.PREFERENCES.ui.useNativeFileChooser currently defaults to false on macOS. Should default to true now likely.

The default value should likely be changed to true on macOS too, ideally, the preferences entry name should be changed to override false on existing macOS systems

Reply:
Fixed